### PR TITLE
diagrams text edits

### LINKED ISSUE #968: Verify the About Graphics Text still make sense
The graphical updates have been approved, but make sure the text aligns with what is in the graphics now.

### PR DESCRIPTION
**Pull request by garemoko**
Fixes #968.

Possibly conflicts #961 and should be merged after.

### CODE REVIEW

**Comment by andyjohnson** [comment]
Thanks for this.  Nice catch on the "may"  +1 and will wait on the About Grammar check to go in first.

File: xAPI-About.md [hunk 1]
```diff
@@ -441,21 +441,22 @@ This section explains and shows graphically how different pieces of xAPI can fit
...
 
+###### Figure 1: Data Flow in xAPI
 ![Data Flow in xAPI](/xAPIDataFlow.jpg)
 
 Figure 1 shows the tracking of learning experiences. A learner has a Learning Experience. This experience could take place in an 
 online course, it could be on the job or it could be part of recreation. It really could be anything. This experience is tracked, 
-on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider may also be responsible for the 
-trusted relationship between the experience and the learner, and perhaps even launching “content” for the learner and digital 
-rights associated with the content, but shouldn't be viewed as boundaries or even "the norm".  
+on the learner’s behalf, by a trusted Learning Record Provider. The Learning Record Provider can also be responsible for the
```
Comment (oconnetf) [inline xAPI-About.md]: This makes me wonder about something that's probably outside scope, here, but... do we not expect "Learning Record Provider" and "Learning Record Consumer" to become abbreviated as "LRP" and "LRC", respectively?

Comment (garemoko) [inline xAPI-About.md]: Yes. We've used those in part 1 of the document at least.

Comment (garemoko) [inline xAPI-About.md]: In fact, in the images in the very section!

**Comment by andyjohnson** [comment]
This did end up with a conflict

**Comment by garemoko** [comment]
@andyjohnson conflict fixed. I don't think I overwrote any of your changes to those two paragraphs, but please do check. Thanks!

**Comment by ljwolford** [comment]
+1 provided it isn't overwriting any of Andy's changes.

**Pull request merged**